src/routes/components/components.json

```
@@ -1,4 +1,77 @@
 [
+	{
+		"title": "sveltekit-adapter-auto",
+		"url": "https://github.com/sveltejs/kit/tree/master/packages/adapter-auto",
+		"description": "Automatically chooses adapter for your current environment if possible",
+		"npm": "https://www.npmjs.com/package/@sveltejs/adapter-auto",
```

The `npm` node is only the NPM package name, not the URL

```suggestion
		"npm": "@sveltejs/adapter-auto",
```